public/css/icinga/widgets.less

```diff
@@ -231,6 +231,10 @@ form.role-form {
     font-style: normal;
   }
 
+  .control-label > * {
+    display: inline-block;
```

Comment:
Okay. That results in Safari breaking on the zero-width-space? But it's the `em`s which get this now, they don't contain the zero-width-spaces at all...

Please explain this. :sweat_smile: